Expose the event types supported by mousetrap

scottwittenburg opened the pull request:
Mousetrap supports passing a third argument, the event type, to the bind function.  This change makes that functionality availability to users of `vue-utilities`.

BryonLewis commented:
Sorry someone must of gave me the capability to approve this since I initially looked at it.  I don't remember getting any notification about it.

scottwittenburg commented:
> Sorry someone must of gave me the capability to approve this since I initially looked at it. I don't remember getting any notification about it.

I already moved all the code from this repo into the miqa application itself, so I'll just close this.  Thanks for approving though, it's good to know the code I moved into miqa make sense.
